source/js/buyers-guide/related-articles.js

```diff
@@ -0,0 +1,21 @@
+const RelatedArticles = {
+  init: () => {
+    const firstParagraph = document.querySelector(
+      ".paragraph-block .rich-text"
+    );
+    const relatedContainer = document.querySelector("#pni-related-content");
+    if (!firstParagraph || !relatedContainer) return;
+    const relatedContent = relatedContainer.querySelector("div");
+    const relatedPlace = firstParagraph.querySelector("*:nth-child(3)");
+    if (!relatedPlace) return;
+    firstParagraph.insertBefore(relatedContent, relatedPlace);
+    relatedContent.classList.add(
+      "large:tw-float-right",
+      "tw-p-4",
+      "tw-pr-0",
+      "large:tw-mr-[-20%]"
+    );
+  },
+};
+
+export default RelatedArticles;
```

Comment:
Thanks for moving this out of the buyer's guide main function. This is definitely nicer.